modules/blocks.js

```diff
@@ -879,11 +879,7 @@ Blocks.prototype.loadBlocksFromPeer = function (peer, cb) {
 			} else {
 				var id = (block ? block.id : 'null');
 
-				library.logger.debug(['Block', id].join(' '), err.toString());
-				if (block) { library.logger.debug('Block', block); }
-
-				library.logger.warn(['Block', id, 'is not valid, ban 10 min'].join(' '), peer.string);
-				modules.peers.state(peer.ip, peer.port, 0, 600);
+				library.logger.debug ('Block processing failed [blocks]', { id: id, err: err.toString (), block: block});
```

Comment:
As discussed we are currently documenting / standardising on code style. Therefore, please fix code styling as @MaciejBaj described.

Also note: `{ 'id': id, 'err': err.toString (), 'block': block }` within code base would normally like so: `{id: id, err: err.toString(), block: block}`. Using literal property names instead of strings. No space between function arguments or braces.

This applies to whole PR.